### PR TITLE
New node app

### PR DESCRIPTION
**Pull request by njain00**
Built the first endpoint. Some things to revisit (will do in another feature branch around cleaning up the first endpoint):
1. error handling
2. dependency injection
3. better endpoint routes (will read up on how to create route endpoints)

### CODE REVIEW

File: src/api/routes/stockRoute.mjs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import StockApiController from '../controllers/stockController.mjs';
+
+export default class StockRoute {
+
+    setStockRoutes(router) {
```
Comment (HJain01) [inline src/api/routes/stockRoute.mjs:5]: Are all the routes going to be configured here for stocks?

File: src/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "git+https://github.com/NJain0001/trading-bot-api.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/NJain0001/trading-bot-api/issues"
+  },
+  "homepage": "https://github.com/NJain0001/trading-bot-api#readme",
+  "dependencies": {
```
Comment (HJain01) [inline src/package.json:20]: Doesn't dotenv also need to be here?

**Review by HJain01 — CHANGES_REQUESTED** [review]

File: src/api/routes/stockRoute.mjs (hunk 1 above)
Comment (njain00) [reply src/api/routes/stockRoute.mjs:5]: the routes will be set here, and the controllers will handle the logic associated to the routes

File: src/package.json (hunk 2 above)
Comment (njain00) [reply src/package.json:20]: added the missing package to the json file

**Review by HJain01 — APPROVED** [review]

**Pull request merged**